### ISSUE TITLE
Typos on Alt Text Article

### ISSUE THREAD

**Issue by harrybailey** [a11yproject/a11yproject.com#96]
"assitive techonology" should be "assistive technology".

Last list item should to end with a full stop.

http://a11yproject.com/posts/alt-text/

I hope that's helpful!
